tools/xray_to_optrack/xray_to_optrack.cxx

```diff
@@ -0,0 +1,153 @@
+#include <llvm/ADT/DenseMap.h>
```

Comment:
Needs a copyright notice if it's in the standard distribution.

Reply:
f88f593